frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -10,15 +10,17 @@
 import { OpPatch } from 'json-patch';
 import _ from 'lodash';
 import { decodeToken } from 'react-jwt';
-import helpers from '../../helpers';
-import { getHeadlampAPIHeaders } from '../../helpers';
+import helpers, { getHeadlampAPIHeaders, isDebugVerbose } from '../../helpers';
 import store from '../../redux/stores/store';
 import { getToken, logout, setToken } from '../auth';
 import { getCluster } from '../util';
 import { ResourceClasses } from '.';
 import { KubeMetadata, KubeMetrics, KubeObjectInterface } from './cluster';
 import { KubeToken } from './token';
 
+// This module has verbose debugging sprinkled throughout.
+// onDebugVerbose('k8s/apiProxy');
```

Comment:
If this is here so it can be uncommented, then maybe part of the comment above it should be "Uncomment the following line to enable verbose debugging."